platformio/src/locales/locale_de_DE.cpp

```diff
@@ -0,0 +1,295 @@
+/* German locale data for esp32-weather-epd.
+ * Copyright (C) 2022-2023  Luke Marzen
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "config.h"
+
+#ifdef LOCALE_de_DE
+
+#include <vector>
+#include <Arduino.h>
+#include "_locale.h"
+
+// LC_TIME
+// locale-based information,
+// see https://man7.org/linux/man-pages/man3/nl_langinfo.3.html for more info.
+// Note to Translators:
+//   The LC_TIME definitions are included in the localedata/locales directory of
+//   the glibc source tree, in files named after the locale codes and encoding
+//   schemes. For example, the en_US locale data is stored in the
+//   localedata/locales/en_US file, which contains the definitions for date and
+//   time formats, month and day names, and other time-related settings for the
+//   LC_TIME category.
+//   https://www.gnu.org/software/libc/sources.html
+// D_T_FMT                 string for formatting date and time
+const char *LC_D_T_FMT     = "%a %d %b %Y %T %Z";
+
+// LC_D_FMT                date format string
+const char *LC_D_FMT       = "%d.%m.%y";
+// T_FMT                   time format string
+const char *LC_T_FMT       = "%T";
+// T_FMT_AMPM              a.m. or p.m. time format string
+const char *LC_T_FMT_AMPM  = "";
+// AM_STR                  Ante Meridian affix
+const char *LC_AM_STR      = "";
+// PM_STR                  Post Meridian affix
+const char *LC_PM_STR      = "";
+// DAY_{1-7}               name of the n-th day of the week
+const char *LC_DAY[7]      = {"Sonntag", "Montag", "Dienstag", "Mittwoch", 
+                              "Donnerstag", "Freitag", "Samstag"};
+// ABDAY_{1-7}             abbreviated ame of the n-th day of the week
+const char *LC_ABDAY[7]    = {"So.", "Mo.", "Di.", "Mi.", "Do.", "Fr.", "Sa."};

```

Comment:
LC_ABDAY and LC_ABMON. Generally, the abbreviated names of days and months do not include a period afterward.

Reply:
hmmm you are right according to glibc. I was using this https://www.localeplanet.com/icu/de-DE/ as reference.  
Changed it